### ISSUE TITLE
Disable right drawer

### ISSUE THREAD

**Issue by EbramTawfik** [Dn-a/flutter_inner_drawer#61]
Some pages I want only to use left drawer so I want to disable right drawer. How can I do that?

**Comment by noobloser** [comment]
@EbramTawfik just set the rightChild: null

**Comment by EbramTawfik** [comment]
@noobloser  yes but even when I set the right child to null I still can swipe and I see empty drawer.

**Comment by Fatimamostafa** [comment]
> @noobloser yes but even when I set the right child to null I still can swipe and I see empty drawer.

Remove the `rightAnimationType` attribute from `InnerDrawer`.
